src/tests.rs

```diff
@@ -39,6 +39,7 @@ fn test_source() {
 }
 
 #[test]
+#[ignore]
```

Comment:
Ignored because during a git bisect I couldn't find a time this succeeded anywhere in git history.

@toumorokoshi What's desired behavior of this? Can you think of sourced files that we need `--completion` for?

Reply:
My recommendation is code is behaving as desired and test describes undesired behavior.

Reply:
(deleted but can revisit)

Reply:
source completion is important, since even if the final execution ends up being sourced, you still want to be able to provide auto-completion for things like arguments to an eventually sourced function.

one example is [a shortcut to a directory where you want to allow tab-completion of sub-directories and files](https://github.com/toumorokoshi/ytlaces/blob/master/files/cookbook/w).

Reply:
but I heard the test has been failing for all of history though. I think it's fine to merge this in deleted to fix the build, then pick up a follow-up PR (which I can own) to merge it back in passing.

Reply:
Addressed ⚡